src/templates/layouts/healthCareLocalFacility/FacilitySocialLinks.tsx

```diff
@@ -0,0 +1,187 @@
+import React from 'react'
+
+interface SocialLink {
+  uri: string
+  title: string
+}
+
+interface OperatingStatus {
+  url: string
+}
+
+export interface FacilitySocialLinksProps {
+  regionNickname: string
+  fieldGovdeliveryIdEmerg?: string
+  fieldGovdeliveryIdNews?: string
+  fieldOperatingStatus?: OperatingStatus
+  fieldFacebook?: SocialLink
+  fieldTwitter?: SocialLink
+  fieldFlickr?: SocialLink
+  fieldInstagram?: SocialLink
+  fieldYoutube?: SocialLink
+}
+
+const FacilitySocialLinks = ({
+  regionNickname,
+  fieldGovdeliveryIdEmerg,
+  fieldGovdeliveryIdNews,
+  fieldOperatingStatus,
+  fieldFacebook,
+  fieldTwitter,
+  fieldFlickr,
+  fieldInstagram,
+  fieldYoutube,
+}: FacilitySocialLinksProps) => {
+  const showGovDelivery =
+    !!fieldGovdeliveryIdEmerg ||
+    !!fieldGovdeliveryIdNews ||
+    !!fieldOperatingStatus
+
+  const showSocialLinks =
+    !!fieldFacebook ||
+    !!fieldTwitter ||
+    !!fieldFlickr ||
+    !!fieldInstagram ||
+    !!fieldYoutube
+
+  if (!showGovDelivery && !showSocialLinks) {
+    return null
+  }
```

Comment:
This is also new. I'm assuming we shouldn't show a "Get updates from..." section with no links in it. 😅